src/js/plugin/sdk/utils.js

```diff
@@ -0,0 +1,11 @@
+
+export default class Utils {
+
+  static addLeadingSlashIfNedded(value) {
```

Comment:
Let's not create a new util only to add a slash.

Reply:
Sure!